test/fastly/client_test.rb

```diff
@@ -115,6 +115,9 @@
         '/service/blah',
         {include_auth: false}
       )
+
+      assert_equal resp.class, Hash
+      assert_includes resp, "i"
```

Comment:
I see other tests are making this assertion, but what does it mean? I see `JSON.generate(i: "dont care")` above, maybe that's why we're asserting `"i"` is present?

Reply:
Good catch, this was an unintentional addition, which I removed in my latest commit.